### ISSUE TITLE
Verify platforms exist for commands you will run

### ISSUE THREAD

**Issue by mlynch** [ionic-team/trapeze#64]
Right now we don't check if android or ios platforms exist before running commands, resulting in some errors that are hiding the actual issue due to optional chaining

**Comment by mlynch** [comment]
I believe this has been resolved

**Comment by AlexandrZl** [comment]
@mlynch looks like it's an issue again since 5.0.0-next.1. If you add

```
android:
    xml:
      - file: app/src/main/res/values/strings.xml
        target: /resources
        merge: |
          <resources>
            <string name="app_name">$PRODUCT_NAME</string>
            <string name="title_activity_main">$PRODUCT_NAME</string>
            <string name="package_name">$ANDROID_PACKAGE_NAME</string>
            <string name="custom_url_scheme">$ANDROID_PACKAGE_NAME</string>
          </resources>
```

and try to run `npx trapeze run config.yaml -y` without android platform you will see this error
```

Fatal error: Error running command
[capacitor] Error: No such XML file for xml operation: strings.xml
```

I guess this was caused the fix you added recently in #87
